docs/index.rst

```diff
@@ -23,13 +23,14 @@ Main Features
 
 .. toctree::
    :maxdepth: 2
-   :caption: User Guide
+   :caption: Getting started
```

Comment:
We've mostly used Title Case not Sentence case, so let's stick to it for consistency (or change it everywhere).

```suggestion
   :caption: Getting Started
```